src/Forms/Label.tsx

```diff
@@ -74,9 +80,16 @@ const StyledLabel = styled.label<StyledLabelProps>`
   grid-template-areas: ${({ labelPosition, isLabelVisible }) => (
     generateGrid(labelPosition, isLabelVisible)
   )};
-  margin: ${fromTheme('ws', 'small')};
+  margin: ${fromTheme('ws', 'small')}
   align-items: baseline;
   gap: ${({ theme }) => (theme.ws.xsmall) + ' ' + (theme.ws.xsmall)};
+  ${({ hideError }) => (hideError
+    ? `margin: 0px;
```

Comment:
I think that we would still want to have the `margin` for the label in the case that `hideError` is true. 
![image](https://user-images.githubusercontent.com/29589689/96283157-c5f52880-0fa9-11eb-8844-0333d6de4c7e.png)

Reply:
OK will take care of this